frontend/src/metabase/query_builder/actions/core/updateQuestion.unit.spec.ts

```diff
@@ -452,7 +458,12 @@ describe("QB Actions > updateQuestion", () => {
 
           await setup({ question });
 
-          expect(loadMetadataSpy).not.toHaveBeenCalled();
+          // models always call this once, when fetching themselves as a card table
+          if (question.isDataset()) {
+            expect(loadMetadataSpy).toHaveBeenCalledTimes(1);
+          } else {
+            expect(loadMetadataSpy).not.toHaveBeenCalled();
+          }
```

Comment:
Once we set up `eslint-plugin-jest`, conditional assertions will become an error. Could you please break this test into two (for model and for other questions)?